rofication/_dbus.py

```diff
@@ -3,15 +3,15 @@
 
 from dbus import service, SessionBus
 from dbus.mainloop.glib import DBusGMainLoop
-from gi.repository.GLib import MainLoop
+from gi.repository.GLib import MainLoop, timeout_add_seconds
 
 from ._metadata import ROFICATION_VERSION, ROFICATION_NAME, ROFICATION_URL
 from ._notification import Notification, Urgency
 from ._queue import NotificationQueue
 from datetime import datetime
 
-NOTIFICATIONS_DBUS_INTERFACE = 'org.freedesktop.Notifications'
-NOTIFICATIONS_DBUS_OBJECT_PATH = '/org/freedesktop/Notifications'
+NOTIFICATIONS_DBUS_INTERFACE = "org.freedesktop.Notifications"
```

Comment:
I'm not sure about the single quote vs double.  Does this change make a syntactic difference or simply style?

Reply:
It's style, but it comes from here:

https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html

Black is an opinionated Python formatter - quite in use these days, as it solves all the religious format wars by imposing one style :-) 

So just running black on a set of files will give you consistency across the code.